src/main/java/org/jabref/gui/maintable/OpenUrlAction.java

```diff
@@ -62,6 +62,10 @@ public void execute() {
                 link = entry.getField(StandardField.URL);
                 field = StandardField.URL;
             }
+            if (entry.hasField(StandardField.ISBN)) {
+                link = entry.getField(StandardField.ISBN);
+                field = StandardField.ISBN;
+            }
```

Comment:
Please move that before the DOI checking (line 57). The DOI should take precedence over the ISBN

Reply:
addressed